calliope.sample-config.js

```diff
@@ -1,62 +1,195 @@
 /**
  * @file
- * Configuration objects used in Gulp tasks.
+ * Configuration objects used by Calliope.
  */
 
-// Paths used throughout our configuration objects.
+/**
+ * paths Object
+ *
+ * Paths used throughout the sample configuration values below. This has no
+ * significance beyond this file; it is merely here to avoid duplication of
+ * source and destination paths in the configuration values used below.
+ */
 const paths = {
-  // source
+  // Source.
   SRC: 'src',
-  // destination
+  // Destination.
   DEST: 'build',
-  // settings
-  CONF: 'toolchain',
-}
+};
 
-// Export a register of assets pipelines: the `build` and `watch` tasks use
-// this object to programmatically run each pipeline. If it aint here, it ain’t
-// getting built! Destination directories are relative to theme root.
+/**
+ * pipelines Object
+ *
+ * A register of assets pipelines: the `build` and `watch` tasks use this
+ * object to programmatically run each pipeline. If it ain’t here, it ain’t
+ * getting built! Destination directories are relative to the current working
+ * directory.
+ */
 exports.pipelines = {
   fonts: {
-    src: [
-      `${ paths.SRC }/fonts/**/*`,
-    ],
-    dest: `${ paths.DEST }/fonts`,
+    /**
+     * src - String or Array of Strings
+     *
+     * A value representing one or more font files to be used as a source. It
+     * may be a path to a single file or a glob matching one or more files, or
+     * an array of said strings.
+     *
+     * Uncomment the next few lines and edit them as necessary. */
+    // src: [
+    //   `${ paths.SRC }/fonts/**/*`,
+    // ],
+
+    /**
+     * dest - String
+     *
+     * The directory to which font files will be moved.
+     *
+     * Uncomment the next line and edit it as necessary. */
+    // dest: `${ paths.DEST }/fonts`,
   },
   images: {
-    src: [
-      `${ paths.SRC }/images/**/*`,
-    ],
+    /**
+     * src - String or Array of Strings
+     *
+     * A value representing one or more image files to be used as a source. It
+     * may be a path to a single file or a glob matching one or more files, or
+     * an array of said strings.
+     *
+     * Uncomment the next few lines and edit them as necessary. */
```

Comment:
```suggestion
     * Uncomment the next few lines and edit them as necessary.
     */
```
I'm curious why JS formats this differently from PHP comments. I've always seen it the way I suggested, but that is with PHP. Does JS prefer the way you have it? 🤷🏼

Reply:
> That said, maybe I should reformat these to the standard Docblock/JSDoc format and trust developers to understand that by “next line” we mean the line after the block is closed.

That has my vote FWIW.

Reply:
Done! See 33d2276.